frame/elections-phragmen/src/lib.rs

```diff
@@ -185,8 +185,12 @@ decl_storage! {
 				// Note: all members will only vote for themselves, hence they must be given exactly
 				// their own stake as total backing. Any sane election should behave as such.
 				// Nonetheless, stakes will be updated for term 1 onwards according to the election.
-				<Members<T>>::append(&[(member.clone(), *stake)])
-					.expect("Failed to append genesis members.");
+				Members::<T>::mutate(|members| {
+					match members.binary_search_by(|(a, _b)| a.cmp(member)) {
+						Ok(_) => (),
```

Comment:
Should I panic here?

Could prevent someone form launching a chain with a bad genesis file

Reply:
How about doing them a favour and sorting it for them? but all in all I am fine with both. although, I assume for humans it is kinda hard to sort account ids. So my preference is sorting whatever is given at genesis.

Reply:
Its not about sorting it. That I do for them. It is about having a duplicate member value in the genesis. I can either:

1. Do nothing if I see a second copy of the same member.
2. Overwrite the first copy with the second copy.
3. Panic and say there is a duplicate member in the genesis.

Reply:
Panic.